### ISSUE TITLE
Why VACUUM udd before DROP?

### ISSUE THREAD

**Issue by nyov** [paulproteus/public-udd-mirror#14]
For curiosity's sake, is there a reason for vacuuming the old db before dropping it?

https://github.com/paulproteus/public-udd-mirror/blob/e2e3b628913b39ac32d939c5727d07d0554a2f8e/scripts/update_udd.sh#L61

https://github.com/paulproteus/public-udd-mirror/blob/e2e3b628913b39ac32d939c5727d07d0554a2f8e/scripts/update_udd.sh#L60-L68

And THANKS for this mirror!

**Comment by mapreri** [comment]
That's a good point.

I just added them in a single moment, without much thinking.  The machine hosting udd-mirror.debian.net back then kept going OOM with many auto-vacuum processes that fought over the few MB or RAM and the single CPU with the incoming connections, so it was just trashing.  Given that the content of the database is not changing except at import time I decided that disabling the auto-vacuum and manually vacuum it once would be great enough, and so I did.

I suppose there isn't much point indeed in vacuuming a database that is about to be dropped though…
